CHANGELOG.md

```diff
@@ -236,6 +236,12 @@ previous `v1.3.x` releases to this release.
   seconds has passed to enforce rate limiting, now instead it configures a
   non-blocking [time.Ticker][go-tick] that runs at the provided seconds
   interval. Input remains a float that indicates seconds.
+- the `KongClusterPlugin` resource now requires an ingress class to be configured
+  (e.g. the annotation `kubernetes.io/ingress.class`) in order to be configured.
+  Historical `KongClusterPlugin` resources which do not have this annotation added
+  before an upgrade may fail with `no KongPlugin or KongClusterPlugin was found` so
+  be sure to add this annotation to your existing `KongClusterPlugin` objects
+  before upgrading.
```

Comment:
[The 1.3.x docs](https://docs.konghq.com/kubernetes-ingress-controller/1.3.x/concepts/custom-resources/#kongclusterplugin) say:

> This resource requires the kubernetes.io/ingress.class annotation.

What is the actual breaking change here?

Reply:
I think I just missed where this was in the docs originally, so this isn't needed.